source/android/Renaissance/lib/create_path.dart

```diff
@@ -298,51 +481,38 @@ class _CreatePathPageState extends State<CreatePathPage> {
 
-                      List<Map<String, String>> milestones = [];
+                      List<Map<String, String>> items = [];
                       List<Map<String, String>> topics = [];
 
                       for (var i = 0; i < _titleControllers.length; i++) {
-                        milestones.add({
+                        items.add({
                           "title": _titleControllers[i].text,
-                          "body": _descControllers[i].text
+                          "body": _descControllers[i].text,
+                          "type": _typeOfController[i],
                         });
                       }
-                      print(milestones);
+                      print(items);
 
                       List<String> splitted = topicController.text.split(",");
 
-                      List<Tag> topicsSubmit = [];
-                      for (var item in splitted) {
-                        List<Tag> resultTags =
-                            await HttpService.shared.searchTopic(item.trim());
-
-                        resultTags = [resultTags[0]];
-                        topicsSubmit = topicsSubmit + resultTags;
-                      }
-
-                      for (var i = 0; i < topicsSubmit.length; i++) {
-                        print(topicsSubmit[i].name);
-                        print(topicsSubmit[i].description);
-                      }
-
                       List<Map<String, Object>> sendTopic = [];
-                      for (var i = 0; i < topicsSubmit.length; i++) {
+                      for (var i = 0; i < _topics.length; i++) {
                         sendTopic.add({
-                          "ID": int.parse(topicsSubmit[i].id as String),
-                          "name": topicsSubmit[i].name as String,
-                          "description": topicsSubmit[i].description as String
+                          "ID": int.parse(_topics[i].id as String),
+                          "name": _topics[i].name as String,
+                          "description": _topics[i].description as String
                         });
                       }
 
-                      User response = await HttpService.shared.createPath(
-                        titleController.text,
-                        descriptionController.text,
-                        milestones,
-                        _image == null
-                            ? ""
-                            : FileConverter.getBase64StringFile(_image),
-                        sendTopic,
-                      );
+                      // User response = await HttpService.shared.createPath(
+                      //   titleController.text,
+                      //   descriptionController.text,
+                      //   items,
+                      //   _image == null
+                      //       ? ""
+                      //       : FileConverter.getBase64StringFile(_image),
+                      //   sendTopic,
+                      // );
```

Comment:
remove commented out code

Reply:
This part is for backend connection and it is a framework for the future implementation. This lines will be implemented before the production code. So I will not remove them yet.